product_supplierinfo_discount/README.rst

```diff
@@ -47,6 +47,7 @@ Contributors
 ------------
 
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Ignacio Ibeas <ignacio@acysos.com>
 
```

Comment:
The discount always applies whatever the setup of your pricelist ("Supplier Prices on the product form", etc.).

(NB: dont make a list of one item)